src/esp/assets/PTexMeshData.cpp

```diff
@@ -678,12 +725,18 @@ void PTexMeshData::uploadBuffersToGPU(bool forceReload) {
 }
 
 PTexMeshData::RenderingBuffer* PTexMeshData::getRenderingBuffer(int submeshID) {
-  ASSERT(submeshID >= 0 && submeshID < renderingBuffers_.size());
+  CORRADE_ASSERT(submeshID >= 0 && submeshID < renderingBuffers_.size(),
+                 "PTexMeshData::uploadBuffersToGPU: the submesh ID "
+                     << submeshID << " is out of range.",
+                 nullptr);
   return renderingBuffers_[submeshID].get();
 }
 
 Magnum::GL::Mesh* PTexMeshData::getMagnumGLMesh(int submeshID) {
-  ASSERT(submeshID >= 0 && submeshID < renderingBuffers_.size());
+  CORRADE_ASSERT(submeshID >= 0 && submeshID < renderingBuffers_.size(),
+                 "PTexMeshData::uploadBuffersToGPU: the submesh ID "
+                     << submeshID << " is out of range.",
```

Comment:
```suggestion
                     << submeshID << "is out of range.",
```

Neither with `Debug` / `Warning` / `Error` / `Fatal` nor with the assert macros you need to wrap values with spaces, that's done automatically to save you extra typing. Putting the space there would make it doubly-spaced. [Docs.](https://doc.magnum.graphics/corrade/classCorrade_1_1Utility_1_1Debug.html)

Same in all other cases ;)